scripts/import/import_EPMC.pl

```diff
@@ -993,9 +1005,253 @@ sub get_current_citations{
     return \%citations;
 }
 
+###
+# Get citations from phenotype tables
+
+sub process_phenotype_feature {
+  my $dba = shift;
+  my $var_ad = shift;
+  my $pub_ad = shift;
+  my $source_ad = shift;
+  my $citation_attribs = shift;
+
+  ## Get studies from phenotype_feature
+  my $attrib_ext_sth = $dba->dbc()->prepare(qq[ select s.study_id, s.source_id, s.external_reference, s.study_type
+                                                from study s 
+                                                inner join phenotype_feature p on s.study_id = p.study_id
+                                                where p.type = 'variation' and p.study_id is not null and s.external_reference is not null
+                                                group by s.study_id, s.source_id, s.external_reference, s.study_type ]);
+  $attrib_ext_sth->execute()||die;
+  my $data = $attrib_ext_sth->fetchall_arrayref();
+
+  my $rsid_sth = $dba->dbc()->prepare(qq[ select object_id
+                                          from phenotype_feature 
```

Comment:
It would be quicker to return the variant name from the phenotype feature when pulling out the study info, rather than query the databases individually for each variant. You can create a hash structure keyed on something unique such as the study id to collect all the info together.